build/includes/website.mk

```diff
@@ -23,12 +23,14 @@
 #
 # Website targets
 #
+UID := $(shell id -u)
```

Comment:
Nice! You could also use the below variation as well.
`UID != id -u`
`GID != id -g`

Reply:
I apologize for the delayed response. Your suggestion seems reasonable. May we proceed with your recommendation?